website/docs/r/eventgrid_event_subscription.html.markdown

```diff
@@ -97,6 +99,16 @@ A `storage_queue_endpoint` supports the following:
 
 ---
 
+An `azure_function_endpoint` supports the following:
+
+* `id` - (Required) Specifies the Azure Resource ID that represents the endpoint of the Azure function where the Event Subscription will receive events.
```

Comment:
we intentionally avoid the phrase `Resource ID` in Terraform since it's ambiguous (and Azure:TF doesn't always map 1:1) - could we make this:

```suggestion
* `id` - (Required) Specifies the ID of the Function App where the Event Subscription will receive events.
```

Reply:
Ok. But with the term `Function` instead of `Function App`, as this must be referenced here.